src/erasure.rs

```diff
@@ -507,8 +525,14 @@ mod test {
         blob_recycler: &BlobRecycler,
         offset: usize,
         num_blobs: usize,
-    ) -> (Vec<Option<SharedBlob>>, usize) {
-        let mut window = vec![None; 32];
+    ) -> Vec<WindowSlot> {
+        let mut window = vec![
+            WindowSlot {
```

Comment:
Should we have a WindowSlot::default() method to do this?

Reply:
yes, thanks.